code/modules/organs/organ_external.dm

```diff
@@ -161,8 +156,16 @@
 	brute *= brute_mod
 	burn *= burn_mod
 
+	// Threshold needed to have a chance of hurting internal bits with something sharp
+	var/const/sharp_thresh_int_dmg = 5
+	// Threshold needed to have a chance of hurting internal bits
+	var/const/thresh_int_dmg = 10
+	// Probability of taking internal damage from sufficient force, while otherwise healthy
+	var/const/dmg_prob = 5
+	// Probability of taking internal damage when hit while lacking encasing bones
+	var/const/no_bone_dmg_prob = 30
```

Comment:
Welllll I mainly just made these so I wouldn't have magic numbers and it would be super easy to tweak things as wanted

#defines seem super clunky, I'll just inline these, then